backend/src/exceptions/invalid-slc-function-error/invalid-slc-function-error.ts

```diff
@@ -0,0 +1,17 @@
+import { ExceptionMessage, HttpCode } from '~/common/enums/enums';
+
+const DEFAULT_MESSAGE = ExceptionMessage.FUNCTION_NAME_EXISTS;
+
+class InvalidSLCFunctionError extends Error {
```

Comment:
```suggestion
class SLCError extends Error {
```
don't forget about file name